src/explorer/pagerankTable/aggregate.test.js

```diff
@@ -9,42 +9,47 @@ import {
   aggregateFlat,
   aggregationKey,
 } from "./aggregate";
+import type {NodeType} from "../../analysis/types";
 
 describe("explorer/pagerankTable/aggregate", () => {
   // TODO: If making major modifications to these tests, consider switching
   // from the hand-maintained connections and types, and instead use the demo
   // adadpters from app/adapters/demoAdapters
   function example() {
-    const nodes = {
+    const nodes: {[string]: NodeAddressT} = {
```

Comment:
These can be read-only: `{+[string]: NodeAddressT}` and `{+[string]: NodeType}`.